website/content/docs/enterprise/index.mdx

```diff
@@ -8,22 +8,34 @@ description: >-
 
 # Consul Enterprise
 
-[Consul Enterprise](https://www.hashicorp.com/consul) features address the organizational complexities of collaboration, operations, scale and governance. It includes capabilities that improve failure resilience, read scalability, and managing access in accordance with organizational policies. It also adds support for application and service architectures across complex network topologies. If you have already purchased Consul Enterprise, please see the [licensing section](#licensing) below.
+[Consul Enterprise](https://www.hashicorp.com/products/consul/)
+features address the organizational complexities of collaboration, operations, scale, and governance
+and are [available in several forms of Consul Enterprise](/docs/enterprise/feature-availability).
```

Comment:
```suggestion
features address the organizational complexities of collaboration, operations, scale, and governance
and are available in several forms of Consul Enterprise. Refer to the [feature availability matrix](/docs/enterprise/feature-availability) for details.
```
It's a (slightly) better user experience to give an idea where you are sending readers.

Reply:
Done